report_thesis/src/sections/proposed_approach/model_selection.tex

```diff
@@ -0,0 +1,103 @@
+\subsection{Model and Preprocessing Selection}
+Having established the validation and testing procedures, as well as, the evaluation metrics in our experiments, this section will detail our approach to model and preprocessing selection.
+
+Choosing the right models and preprocessing techniques for \gls{libs} data analysis is a challenging task. 
+As the literature highlighted in Section~\ref{sec:related-work} suggests, a variety of models and preprocessing techniques that promise to be adept at handling data that exhibits high-dimensionality, multi-collinearity, and matrix effects.
+Additionally, different machine learning models perform better on some oxides than others. 
+These challenges and model-specific strengths suggests that an optimal approach would involve combining multiple models. 
```

Comment:
I _really_ like the flow of this section, until around this point. I think it's imperative that we provide strong rationale for why we believe this approach would work. Since we switch to focus on the strengths of stacking after this part, I don't believe we're quite at a strong rationale yet.
And here I'm referring to providing a strong rationale for our overall strategy, because you did quite well at doing so for the models we've selected.

See my proposal!